src/app/home-page/home-logged-out/home.component.scss

```diff
@@ -118,3 +118,10 @@ h3 {
 .warn-1 {
   color: mat-color($dockstore-app-warn, darker);
 }
+
+// Style home search bar
```

Comment:
if you can't figure out how to not have this still, put a TODO along with your best guess on what the problem is